core/builder.go

```diff
@@ -27,6 +27,10 @@ type BuildCfg struct {
 	// If online is set, the node will have networking enabled
 	Online bool
 
+	// If permament then node should run more expensive processes
+	// that will improve performance in long run
+	Permament bool
```

Comment:
Didn't use `Ephemeral` as it would be hard to find all tests and places where node is created.

This part is on my list to refactor for the split to: `Online`, `Offline` and `Ephemeral`.